CMakeLists.txt

```diff
@@ -343,7 +343,7 @@ foreach(kind ${kinds})
                                                  amip_interp/include)
   target_compile_definitions(${libTgt}_f PRIVATE "${fms_defs}")
   target_compile_definitions(${libTgt}_f PRIVATE "${${kind}_defs}")
-  set_target_properties(${libTgt}_f PROPERTIES COMPILE_FLAGS ${${kind}_flags})
+  set_target_properties(${libTgt}_f PROPERTIES COMPILE_FLAGS "${${kind}_flags}")
```

Comment:
just looking at lines 76 - 80, maybe the message should be updated to defaulting to mixed-precision.

Reply:
Also, line 278 - 278, I don't think the OVERLOAD macros  are there anymore